libraries/custom_appbase/tests/custom_appbase_tests.cpp

```diff
@@ -89,7 +89,7 @@ BOOST_AUTO_TEST_CASE( execute_from_read_queue ) {
    // stop application. Use lowest at the end to make sure this executes the last
    app->executor().post( priority::lowest, exec_queue::read_only_trx_safe, [&]() {
       // read_queue should have current function and write_queue should have all its functions
```

Comment:
This comment `read_queue should have current function` and the same ones below should be updated or removed.

Reply:
Updated